schemas/non-atlas/customCoordinateSpace.schema.tpl.json

```diff
@@ -18,10 +18,10 @@
     },    
     "axesOrigin": {
       "type": "array",        
+      "minItems": 2,      
       "maxItems": 3,    
-      "minItems": 2,
       "uniqueItems": true,
-      "_instruction": "Enter the origin of this custom coordinate space (central point where axes intersect; 2D: [x, y] or 3D:[x, y, z]).",
+      "_instruction": "Enter the origin (central point where all axes intersect) of this common coordinate space for two-dimensional spaces as [x, y] or for three-dimensional space as [x, y, z].",
```

Comment:
As this is the custom coordinate space schema, shouldn't the instructions point to custom and not common coordinate space?